src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Application {
+    private static final int FIGURE_COUNT = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        GeometricFigure[] figures = new GeometricFigure[FIGURE_COUNT];
+        for (int i = 0; i < figures.length; i++) {
```

Comment:
Minor: Since you already have FIGURE_COUNT to represent the length of an array, I suggest using that variable in for expression for better readability